src/main/resources/export/cpp/Command-constructor.cpp

```diff
@@ -17,4 +17,5 @@ m_$param.getName() = $param.getName();
 #@autogenerated_code("requires", "    ")
 #parse("${exporter-path}Command-requires.cpp")
 #end
+    SetRunWhenDisabled($command.getProperty("Run When Disabled").getValue());
```

Comment:
Is there any way to have these only output this code if Run When Disabled is true?  Since 99% of the time it's going to be false, this feels like it clutters the code a lot.